src/components/Auth/AuthBtnGroup/AuthBtnGroup.tsx

```diff
@@ -76,6 +78,15 @@ function AuthBtnGroup() {
         textColor="#3D3D3D"
         onPress={() => navigation.navigate('Login')}
       /> */}
+      <ImgButton
+        imgSrc={Images.msftLogo}
+        btnText={`${continueWith} Onboarding`}
+        backgroundColor="#357EC7"
```

Comment:
can we create a color pallete or theme in future?

Reply:
Yes, you are right. Will create a task for this.